### PR TITLE
Change to deal.ii version 9.4.0

### PR DESCRIPTION
**Pull request by slfuchs**
This PR follows [#13553](https://github.com/dealii/dealii/pull/13553) and should fix the currently failing GitHub actions.

Edit: ~~Besides this PR extends the GitHub actions to test not only against the latest deal.ii version but also against version 9.4.0.~~

### CODE REVIEW

**Comment by slfuchs** [comment]
@peterrum Currently there is no `v9.4.0-focal` docker image on https://hub.docker.com/r/dealii/dealii/tags. Do you know if this is to follow?

If not we could just keep the test against the `master-focal` docker image here, or test against `v9.3.0-focal` and also set `v9.3.0` in `CmakeLists.txt`.

File: .github/workflows/tests.yml [hunk 1]
```diff
@@ -20,7 +20,7 @@ jobs:
       fail-fast: false
       matrix:
         build_type: ["Release", "Debug"]
-        dealii_version: ["master"]
+        dealii_version: ["master", "v9.4.0"]
```
Comment (peterrum) [inline .github/workflows/tests.yml]:
```suggestion
        dealii_version: ["master"]
```

**Review by peterrum — COMMENTED** [review]
There will be 9.4 in May/June after the release.

File: .github/workflows/tests.yml (hunk 1 above)
Comment (slfuchs) [reply .github/workflows/tests.yml]: Done.

**Comment by slfuchs** [comment]
Thanks for the quick response. I adapted the PR as proposed.

**Review by peterrum — APPROVED** [review]

**Pull request merged**